static/home/css/style.css

```diff
@@ -492,6 +492,10 @@ nav{
 	transform: rotateX(360deg) translateX(-10px) translateY(-300px);
 	background-color: transparent;
 }
+.glide__bullets{
```

Comment:
Use `#initiatives .glide_bullets` instead of just `.glide_bullets` as we want the change to affect the glide carousel in only `initiatives` section and not on the complete website. 

Also, on doing so, you will no longer need to use `!important` as using `id` as css selector has precedence over using `class`.